playwright/_impl/_locator.py

```diff
@@ -517,6 +517,7 @@ async def screenshot(
         mask: List["Locator"] = None,
     ) -> bytes:
         params = locals_to_params(locals())
+        del params["timeout"]
         return await self._with_element(
             lambda h, timeout: h.screenshot(timeout=timeout, **params)
```

Comment:
```suggestion
            lambda h, timeout: h.screenshot(**params)
```

do you mind adding a test for it? I think also this is a cleaner bugfix.

Reply:
Sure. I can add the test.

But I have one question about your solution:

- The `lambda function` also has an arg: `timeout`. If you remove `timeout=timeout` that means `timeout` arg in the lambda function is uesless. 
  So which (variable) value these methods should revieved?

```py
    async def select_text(self, force: bool = None, timeout: float = None) -> None:
        #                                           [m-arg]
        params = locals_to_params(locals())
        del params["timeout"] # [PR]
        return await self._with_element(
            lambda h, timeout: h.select_text(timeout=timeout, **params), timeout
        #             [f-arg]                        [f-arg?]    [PR]    [m-arg?]
        )

    # m-arg -> method arg
    # f-arg -> lambda function arg
```